### PR TITLE
Simplify dev flow

### PR DESCRIPTION
**Pull request by alexeykazakov**
- [x] `make all` is now the default target
- [x] no need to pass configuration file in dev mode
- [x] improved help target

It's now possible to run a service by simply running:
```
> make dev
```
Or alternatively:
```
> make
> export F8_DEVELOPER_MODE_ENABLED=true
> docker-compose up -d db
> ./bin/env
```

### CODE REVIEW

**Comment by alexeykazakov** [comment]
[test]

File: configuration/configuration_blackbox_test.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	// Explicitly set via AUTH_WIT_URL env var
+	os.Setenv("F8_AUTH_URL", "https://api.some.io")
+	config, err = configuration.New("")
+	require.NoError(s.T(), err)
+	assert.Equal(s.T(), "https://api.some.io", config.GetAuthServiceURL())
+
+	os.Unsetenv("F8_DEVELOPER_MODE_ENABLED")
+	config, err = configuration.New("")
+	require.NoError(s.T(), err)
+	assert.Equal(s.T(), "https://api.some.io", config.GetAuthServiceURL())
```
Comment (MatousJobanek) [inline configuration/configuration_blackbox_test.go:47]: isn't this 3 separated cases? I know that the cases are very simple - it just looks weird having it in one test method and not separated (for example to subtests)...

Comment (alexeykazakov) [reply configuration/configuration_blackbox_test.go:47]: They depend on each other. So, we can't move them to separate functions. They are not independent. Subtests would work here (not parallel though) but IMO it's just cosmetic/code-style thing.

**Review by MatousJobanek — APPROVED** [review]
Looks good :+1:

**Pull request merged**